state/PlayerState.gd

```diff
@@ -3,13 +3,27 @@ extends Node
 class_name PlayerState
 
 signal hull_health_changed
-var position = Vector2(50, 50)
-var hull_health = 100 setget set_hull_health
+var position = Vector2.ZERO
+var hull_health = 100 setget _set_hull_health
 var selection = null
 var character = null
 var ship_type = null
-var ship = null
 
-func set_hull_health(val):
+func _set_hull_health(val):
```

Comment:
Private functions (those not called from other scenes) should be prefixed with a `_`